src/App.tsx

```diff
@@ -1,27 +1,48 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
+type State = {
+  isClockVisible: boolean
+};
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+export class App extends React.Component<{}, State> {
+  state = {
+    isClockVisible: true,
+  };
 
-  // eslint-disable-next-line
-  console.log(timerId);
+  clockVisible = () => {
+    this.setState({ isClockVisible: true });
+  };
 
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+  clockInvisible = () => (this.setState({ isClockVisible: false }));
 
-export default App;
+  render() {
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        <p>
+          Current time:
+          {' '}
```

Comment:
Hmm, does this have some use?

Reply:
it's just a way to place a space so linter is not angry 🙂